### PR TITLE
- reveal icon toggle

### PR DESCRIPTION
**Pull request by MrEssex**

### CODE REVIEW

**Review by bajb — APPROVED** [review]

**Review by TomK — CHANGES_REQUESTED** [review]
as per discussion

File: src/Layout/Reveal/res/index.js [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
   else
   {
     console.warn('Cannot reveal. The target does not exist');
   }
-  if(reveal.hasAttribute('reveal-destructive'))
-  {
-    document.querySelectorAll(`[reveal-launcher="${targetId}"]`)
-            .forEach(ele => ele.parentNode.removeChild(ele));
-  }
+
```
Comment (TomK) [inline src/Layout/Reveal/res/index.js:26]:
```suggestion
```

File: src/Layout/Reveal/res/index.js [hunk 2]
```diff
@@ -1,21 +1,26 @@
 import './reveal.css';
 import {on} from '../../../Foundation/res';
 
-on(document, 'click', '[reveal-launcher]', (e) =>
-{
+on(document, 'click', '[reveal-launcher]', (e) => {
   const targetId = e.delegateTarget.getAttribute('reveal-launcher');
   const reveal = document.querySelector(`#${targetId}`);
```
Comment (TomK) [inline src/Layout/Reveal/res/index.js:6]:
```suggestion
  const reveal = document.getElementById(targetId);
```

**Review by TomK — APPROVED** [review]
lgtm, could you test again with the commit i made pls?

**Pull request merged**